packages/next/build/output/index.ts

```diff
@@ -173,7 +173,13 @@ buildStore.subscribe(state => {
       }
 
       if (Object.keys(amp).length > 0) {
-        warnings = (warnings || []).concat(formatAmpMessages(amp))
+        warnings = warnings || []
```

Comment:
This can most likely achieve the same result without having to do an additional check
```suggestion
        warnings = (warnings || []).concat(formatAmpMessages(amp) || [])
```

Reply:
Thanks for the shorthand of the check with `||`. This works. I have updated the PR.